File: Ileya/Day 1/JavaScript/Level 1/Task1.js
//const splitEvenOdd = function (arr){
function splitEvenOdd(arr){
    let odds = [];
    let evens = [];

    for (let index = 0; index < arr.length; index++) {
        if (arr[index] % 2 == 1) {
            odds.push(arr[index]);
        } else {
            evens.push(arr[index]);
        }
    }
    return [odds, evens];
}

module.exports = splitEvenOdd;

File: Ileya/Day 1/Python/Level 1/task_1.py
def split_even_odd(arr):
    odds = []
    evens = []

    for index in range(len(arr)):
        if arr[index] % 2 == 1: 
            odds.append(arr[index])
        else:
            evens.append(arr[index])
        
    
    return [odds, evens]
